Atlas-class Light Armored Crusier

meemofcourse opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
![imagen](https://github.com/user-attachments/assets/e76c2cc9-9b25-496b-b8ff-a82b3b4debc1)
![atlas2](https://github.com/user-attachments/assets/3ca2427c-e333-4d87-8202-9f81842378b3)

The Atlas-class is the first Colonial League ship following the CLIP Rework. The decaling and general decoration of the ship needs a revision, but otherwise, it is intended as a small, overall generalist patrol vessel to the likes of the Vaquero. The 4-Minutemen team, lead by one Sergeant, is subordinate to the First Officer and Captain, who overlook the operation of the ship by the Corpsman and the Engineer.

It is staffed by:
* 1 Captain
* 1 First Officer
* 1 Sergeant
* 1 Corpsman
* 1 Engineer
* 4 Minutemen
* 1 Deckhand

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
it's the clip ship it uses the colon league content

<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog

:cl:
add: Atlas-class Light Armored Crusier
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

meemofcourse committed:
so cool

meemofcourse committed:
Merge remote-tracking branch 'upstream/master' into clipper-boat

meemofcourse committed:
Merge remote-tracking branch 'upstream/master' into clipper-boat

meemofcourse committed:
Update minutemen_atlas.dmm

meemofcourse committed:
Merge remote-tracking branch 'upstream/master' into clipper-boat

meemofcourse committed:
Merge remote-tracking branch 'upstream/master' into clipper-boat

meemofcourse committed:
i must eat goblins

meemofcourse committed:
kill goblins

kaylexis commented:
CLIP IS SO BACK.

Martinpachu commented:
it's peak chat

Martinpachu commented:
![imagen](https://github.com/user-attachments/assets/159cfdde-70b7-4c97-8264-6fbcb2ca9523)
it has a 762 box but no guns that use 762 i think?

Martinpachu commented:
there's no gravity generator!!

Martinpachu commented:
i might be a stupid baby but i don't think the cargo bay doors are linked to any buttons. also no holofields ? also just my dumb gay opinion but there could be another candor mag

firebudgy commented:
![image](https://github.com/user-attachments/assets/d7fbc149-c88c-46a8-b477-37327339cae7)
I think you forgot a couple wires for the engines here

firebudgy commented:
![image](https://github.com/user-attachments/assets/2a5bc8cf-23b5-49f2-a6b7-404e43c474f9)
Bridge lockdown button closes all these shutters, while the bridge windows button does nothing

![image](https://github.com/user-attachments/assets/5d0cf866-6fe2-411a-a839-4991a362dc55)
Every other ship usually has windows blocking the dorms so SSD people don't get obliterated by carps while away from their computer, so I'm just wanting point it out in case you forgot or something

![image](https://github.com/user-attachments/assets/a3c7531a-0fe7-4817-b3f3-dc9ffc693dbb)
Same with the medical bay external window shutters, I'm just pointing this out in case their omission was an accident

Sadhorizon commented:
- They don't have a single space suit on the entire ship.
![obraz](https://github.com/user-attachments/assets/ee5e56e1-5a1a-4214-a3c3-056aa939c2fe)
- Why is it just... sticking out like that... _surely_ there is a better way to handle it. Maybe a window into grilles?
![obraz](https://github.com/user-attachments/assets/19e89697-5c9a-4329-9e06-b1ccf4b29499)
- Most ships have shutters on the precharger. Cuz, technically, you are blocking engine output rn. (lore-wise)
- The cargo bay has very little cargo space. I'd suggest, on the right side, making more room for a crate shelf. Also switching the air tank rack for an air tank crate, cuz it's less stationary.
![obraz](https://github.com/user-attachments/assets/d1d98293-3f52-4279-82fc-8d6e91bd239d)
- This locker has the wrong dir.
![obraz](https://github.com/user-attachments/assets/0f7dee28-22b9-4449-8f05-ee542b28e21e)
- This airlock has no controller and no vents.
![obraz](https://github.com/user-attachments/assets/7a920856-c165-4f4e-8923-37f1b8029ebc)
- Empty space spotted. Could put a vendor or something here. Cigs never hurt anybody.
![obraz](https://github.com/user-attachments/assets/6522adcc-588e-401e-a40a-73a90cf07663)
- No air pump? Also, missing cable (marked green).
![obraz](https://github.com/user-attachments/assets/eb42e68a-e3d5-45a9-8cca-7225840f3f81)
- You forgot holofields, and also a shutter button.
- Bridge airlocks don't have access req set.
- Airlocks aren't named. Probably not needed, but still nice to have.
![obraz](https://github.com/user-attachments/assets/9fb8e095-8aba-4f47-bf60-b161d1c8e3d6)
- You have two air cans, but no ports to put them in. Would probably be nice.
- All of your light switches are like. Very high. Latency explained to me before, that shadow should be touching the ground. Just pointing it out.
![obraz](https://github.com/user-attachments/assets/21aae4a8-c4ce-4a59-be6d-07fad52e0862)
- When those are next to each other, .i makes you speak on both, which is kinda annoying.
- Medical lacks surgical tools.
- There is no CLIP stamp on board.

Martinpachu commented:
small suggestion, maybe start with a bit of glass and iron for the autolathe?

Martinpachu commented:
![imagen](https://github.com/user-attachments/assets/1cad2242-63a8-4901-8f17-fc2e1c7da27c)
the turret panel is unreachable

Sadhorizon commented:
> small suggestion, maybe start with a bit of glass and iron for the autolathe?

Some plastic is neat to have too! Just a lil bit, for when you need it.

Martinpachu commented:
deleted because im stupig nevm

meemofcourse committed:
competent maptainer

meemofcourse commented:
> ![imagen](https://private-user-images.githubusercontent.com/86135885/352004563-159cfdde-70b7-4c97-8264-6fbcb2ca9523.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE5NDEzMTksIm5iZiI6MTcyMTk0MTAxOSwicGF0aCI6Ii84NjEzNTg4NS8zNTIwMDQ1NjMtMTU5Y2ZkZGUtNzBiNy00Yzk3LTgyNjQtNmZiY2IyY2E5NTIzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzI1VDIwNTY1OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTAwMTBiMDRhOTI1YzM4YjRjNDM2NWNjYjcyNjAyNzFlMzg5YmUyMjNhMzEwMDA1Y2Q1ZDc4YmNjOTQ2MDM4MGUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.wEasLTCmFxQY68AzE2JZIR5QGfSX5KMffDeNfOx2wmo) it has a 762 box but no guns that use 762 i think?

![imagen](https://github.com/user-attachments/assets/34cb35f4-6061-4e73-b56c-cd013954e4b7)
:(

meemofcourse commented:
> Why is it just... sticking out like that... surely there is a better way to handle it. Maybe a window into grilles?

nah
> Empty space spotted. Could put a vendor or something here. Cigs never hurt anybody.

i am avoiding vendors on the argument that this is a military vessel
> There is no CLIP stamp on board.

intended for admin use. you shouldn't be able to stamp things on behalf of the entire country

meemofcourse committed:
I want to kill the Prime Minister of the Zohil Explorat

meemofcourse committed:
ougjjjjhhh

Martinpachu commented:
![imagen](https://github.com/user-attachments/assets/87cb94e8-d396-4d0b-ab0a-34202abb3b9b)
cargo bay doors opens only one door. also, idk if i just didn't see it, but there should be a cell charger somewhere. since that's standard for every ship i believe.

Martinpachu commented:
actually, weren't communications consoles off limits? since you can do stuff like request nuclear access codes and send announecements. i vaguely remember them not being allowed.

meemofcourse committed:
checks

Martinpachu commented:
is the sargeant supposed to go out with the minutemen, as like leading an away squad? if so, they should probably have at least a handgun for themselves.

Martinpachu commented:
sorry for not saying all this stuff all at once by the way. i just realize some stuff later

rye-rice committed:
im sorry meem i need to control who plays it during the uptime at least for now

Signed-off-by: rye-rice <58402542+rye-rice@users.noreply.github.com>

rye-rice committed:
REVERT WHEN DONE: path changes for clip guns, canwebedit work miracles?

Signed-off-by: rye-rice <58402542+rye-rice@users.noreply.github.com>

meemofcourse committed:
Merge remote-tracking branch 'upstream/master' into clipper-boat

meemofcourse committed:
sarge db

meemofcourse committed:
Merge remote-tracking branch 'upstream/master' into clipper-boat

meemofcourse committed:
I FUCKING HATE CAKE

meemofcourse committed:
Merge branch 'shiptest-ss13:master' into clipper-boat

meemofcourse committed:
Merge branch 'master' into clipper-boat

meemofcourse committed:
brig

meemofcourse committed:
Merge branch 'master' into clipper-boat

meemofcourse committed:
Update minutemen_atlas.dmm

meemofcourse committed:
Merge branch 'clipper-boat' of https://github.com/meemofcourse/Shiptest into clipper-boat

goober3 reviewed (dismissed):
hi, i'm 'Shiptest Maptainers' and i am going to review this PR.

![f1f64e81897216c2086df5c143c19bd9](https://github.com/user-attachments/assets/0d879a82-82ab-487b-979f-87b6ea6f7660)
the cryo room could do with a light switch.

![8f7580882b9d75c13bef97899ad81a54](https://github.com/user-attachments/assets/75bee621-0d5c-40c9-a13e-ab03b0745f18)
consider removing the curtain here, i don't think it works with the cyborg recharger sprite.

![9ec48364337401afc8db399a08ebea88](https://github.com/user-attachments/assets/1f6e51f7-6185-4fb7-88dc-aa1e70822494)
please make these sidings reach the bottom of the room i beg

![d479ab47eda7ea6bfbefd70f70e70fc9](https://github.com/user-attachments/assets/41b42829-5dcb-4cea-9bd9-6f7491b933fa)
these railings don't look great in my opinion!!! consider making it a single rail or removing them altogether. also look into accenting the floor in the wider canteen area with some techfloor to make it a little more visually interesting.

![659420d0ee230553c0f6d3bf1b090e5f](https://github.com/user-attachments/assets/2b2e69ae-4e12-40b9-85bc-9e22ce042489)
i think the cell here warps the ship shape a little too much. could you maybe move it to where the grilles are here, or somewhere else on the ship.
![image](https://github.com/user-attachments/assets/e244b035-af59-4a44-a750-45e57c596120)

![ed3e8b83d90f425fcb1fa1a2875042b8](https://github.com/user-attachments/assets/58765300-7952-4d68-8ca3-aea6edd820b7)
i like this hallway. why is there a microwave here. can you put warning stripes on the firelocks and replace them with directional variants by the way.

![ff6c5492a1bbd39a9dd05fca8f034920](https://github.com/user-attachments/assets/48df368a-340b-47fc-a14b-2c7a70d4045b)
i also really like this room. consider giving them a crate shelf (x1)

![image](https://github.com/user-attachments/assets/3e460c0d-8a31-4a13-9559-7e5773cfc2c6)
oh no, i guess i can't perform this life-saving surgery. what a shame.

we have CLIP guns now. use them. look into [code/modules/projectiles/guns/manufacturer/clip_lanchester/ballistics.dm](https://github.com/shiptest-ss13/Shiptest/pull/3238/files#diff-4ef8305cfcce0303b5abd84ec7df5d12446c01acb9d5aa697fdd5c139234e6e4) for an idea of what to use where. (replace the captain's pistol with a CM-70)

![74f62c97f4ea57dbb6c7c5d19cf03eae](https://github.com/user-attachments/assets/9eb9d681-4c71-4336-84ea-0c3732d9bf4e)
guess i'll fly into an asteroid field then.

![image](https://github.com/user-attachments/assets/b9669959-9b9c-42a4-bc44-13a8c88d316e)
why does the first officer have so much clothing

![e7880e40a6e8f5b207444db7eba0afc4](https://github.com/user-attachments/assets/135e0cb4-d50f-457c-9c12-4c87c43380e0)
consider more catwalks and rusted plating on both sides of the engine room. this seems a little too clean and orderly. also, add windoors to the engine chargers.

![6ea8fe7f53523cf87cc90c18210d145c](https://github.com/user-attachments/assets/21b80bd5-eba6-4400-9cc9-3ed0848eab34)
these shutters (bottom) appear to be connected to the dorm windows rather than the FO's privacy shutters

that's about it i think

meemofcourse commented:
> consider removing the curtain here, i don't think it works with the cyborg recharger sprite.

but lat did it in the panacea and nobody said anything :(
> can you put warning stripes on the firelocks and replace them with directional variants by the way.

yes, but i'm not changing them to directionals because this is a door dividing a hallway and the thin ones work best for airlocks
> oh no, i guess i can't perform this life-saving surgery. what a shame.

> guess i'll fly into an asteroid field then.

cdn is fucked what do you want me to do
> why does the first officer have so much clothing

formal uniforms to hand to the crew. you know, it's a military

meemofcourse committed:
Merge remote-tracking branch 'upstream/master' into clipper-boat

meemofcourse committed:
jeorg review

jeorg shiptest maptainers with 30 reviews is an outlier and should not be counted

rye-rice commented:
I will assume by lack of maptainer review this is ready to merge

meemofcourse commented:
please don't until another maptainer approves it

meemofcourse committed:
Merge remote-tracking branch 'upstream/master' into clipper-boat

meemofcourse committed:
RYE STOP REPATHING THINGS

rye-rice commented:
@Latentish @Apogee-dev

meemofcourse committed:
Merge remote-tracking branch 'upstream/master' into clipper-boat

meemofcourse committed:
checks

Sadhorizon commented on _maps/configs/minutemen_atlas.json:
```diff
@@ -0,0 +1,49 @@
+	{
```
Add `"faction": "/datum/faction/clip",` somewhere, or it's gonna show as indie on the spawning menu.

meemofcourse committed:
unadminspawn

meemofcourse committed:
Merge branch 'master' into clipper-boat

meemofcourse committed:
melee repath

also enabled to true because apparently it doesnt work if you leave it empty despite being true by default

meemofcourse committed:
Merge branch 'master' into clipper-boat

Signed-off-by: meem <75212565+meemofcourse@users.noreply.github.com>

meemofcourse committed:
Update portable_turret.dm

sigh

Signed-off-by: meem <75212565+meemofcourse@users.noreply.github.com>

Gristlebee reviewed (commented):
The table in the engineering wing merges together under the railing.
![image](https://github.com/user-attachments/assets/4e4b74c6-6854-484c-8fcf-40e41929f9ea)
![image](https://github.com/user-attachments/assets/6cc9ddec-8e5e-49ae-bce7-98c35d61a763)
Replace the regular box of shotgun shots with an ammo box please.
![image](https://github.com/user-attachments/assets/51c064f6-4909-470d-a3b0-44a38d78421f)
Could you also put the double barrel and CM-23s the officers get into gun cases for consistency?

This is a strong arsenal to be starting with on roundstart. I'd suggest cutting a CM-5s and CM-82s and replacing it with two CM-23 pistols.
![image](https://github.com/user-attachments/assets/cde9238b-8545-414a-9dc6-dd63632decdf)

This is funny, but I'm concerned about them not having a functioning microwave with comparatively few starting rations to the ship's crew size. Maybe put a microwave board in engineering they can put together, or give them some more rations.
![image](https://github.com/user-attachments/assets/f42eb8b6-e637-4eec-a3d3-303d28e75dbd)

I'd also suggest cutting one of the 20 stacks of plasma in the engine room. The ship doesn't need that much fuel to start with I feel.

DIB-DOG commented:
Having some survival knives or any melee weapon really, would be nice for the minutmen as well.

meemofcourse committed:
review

meemofcourse committed:
knifes too ig

Gristlebee reviewed (commented):
Please reduce the amount of extra mags for the CM-82, and CM-5 to two since there's only one of each now. It would also be good to swap out the combat knives for survival knifes to bring it in line with other ships.
![image](https://github.com/user-attachments/assets/20569a66-ef0b-4da3-abe0-2480a2049a36)

Aside from that, everything looks pretty good to me.

meemofcourse committed:
Merge branch 'master' into clipper-boat

Signed-off-by: meem <75212565+meemofcourse@users.noreply.github.com>

Erikafox commented:
recently played on this. here are my thoughts

![image](https://github.com/user-attachments/assets/49aacfb9-81f7-45a9-bbcc-50638783754d)
you should curve the railings, the middle tile that's obviously technology feels weird to stand on. 
![image](https://github.com/user-attachments/assets/2ab13eba-36ea-4589-b3a8-afa0e8615fee)
Honestly thought this was kinda ugly
![image](https://github.com/user-attachments/assets/e7929637-5811-494f-90c1-ed4b1d7ea42d)
don't care for the engines in this style. I think being able to see the terminals is kinda assumed. Maybe put some decalling where they should be? The steeldecal plates are good for that.
First Officer and Captain should have a spare CMM bag (and imo should spawn with one, the captain bag doesn't fit them too well)
![image](https://github.com/user-attachments/assets/91d9e752-ac20-45c4-ac02-05054346872d)
This feels somewhat haphazard
Airlocks all need to be labelled
![image](https://github.com/user-attachments/assets/2fe112f7-b2e0-47b9-8d19-30ab69ef75ce)
this is like. a lot of unused space imo. take some of it and give the captain a nice little front office?
![image](https://github.com/user-attachments/assets/cf200acc-8050-42be-8e45-276176aa5ce7)
spread some folders out on the tables - and put some paperwork stuff in the bridge too.
![image](https://github.com/user-attachments/assets/bd52db32-023d-47ca-be21-d772da235c8c)
this does not look good in game
![image](https://github.com/user-attachments/assets/78c9d063-26f7-4162-ba99-89b83c48e1da)
You may want to secure the blackbox somewhat
![image](https://github.com/user-attachments/assets/c6cd44c1-101a-49b8-a65d-76c9857be05b)
These should be SSU'd. maybe take some cargo bay space (and expand it a bit) to accomodate, or eat into the briefing area.
![image](https://github.com/user-attachments/assets/156a7b37-62fb-4e25-8e03-8ea00381ad3d)
grav gen feels kinda just. tacked on there.

meemofcourse commented:
thats it bitch im coming for you

meemofcourse committed:
Merge remote-tracking branch 'upstream/master' into clipper-boat

meemofcourse commented:
> don't care for the engines in this style

What

> take some of it and give the captain a nice little front office?

no

> put some paperwork stuff in the bridge too.

there is a paper cutter, paper bin, photocopier, clipboard, and stamps i literally dont know what more you want. i'm not adding the clip stamp because standard individual ships should not be able to give the approval of an entire nation

meemofcourse committed:
sighs so deeply

Erikafox commented:
>engines

I just prefer the style of seeing the terminal and having more like.. technical grunge around it

Rest is fair its your ship

meemofcourse committed:
tap tap

meemofcourse committed:
oops

meemofcourse committed:
Update minutemen_atlas.dmm

meemofcourse committed:
BWAAAH

Sadhorizon commented:
please fix this
![obraz](https://github.com/user-attachments/assets/409cfbc4-11c5-4488-99bb-09167f9a43d5)

goober3 commented:
![a8d2c65a1b1debfea67dcc6eb6715715](https://github.com/user-attachments/assets/5c1567c2-23b0-45bf-883e-ac1d442e1231)
just a thought- consider turning this external injector into more of a feature somehow. maybe some decals or railings, think about expanding the platform its on. it feels somewhat haphazard in its placement at the moment. also please fill that corner next to the engine precharger or move the water tank towards it.

meemofcourse committed:
Update minutemen_atlas.dmm

Gristlebee commented:
This PR should be good to go.
